app/shared/user.model.ts

```diff
@@ -1,4 +1,4 @@
-const validator = require("email-validator");
+import * as validator from 'email-validator';
 
 export class User {
   username?: string;
```

Comment:
Lets remove the `username` property of `User` since it is not being used anymore.

Reply:
Should the `email` property not be optional anymore either? It seems like it shouldn't to me.